docs/releasenotes.md

```diff
@@ -1,9 +1,12 @@
----
-layout: page-with-side-nav
-title: Haal Centraal BRP bevragen
----
 # Releasenotes Haal-Centraal-BRP-Bevragen
 
+## Versie 1.2.0
+
+- De property "adellijkeTitelPredikaat" is verplaatst van "NaamPersoon" naar "Naam"
+  Dit heeft als gevolg dat deze property nu ook onderdeel is van de resources "ouder", "kind" en "partner"
+- De property "adellijkeTitelPredikaat" (boolean) is toegevoegd aan "InOnderzoek" van het schemacomponent "Naam"
+- Bij "NaamPersoon" is de boolean "aanduidingNaamGebruik" toegevoegd aan "inOnderzoek"  
```

Comment:
Het ene moment benader je het puur vanuit schemacomponenten (vooral 1e punt), en het andere moment vanuit de gegevens in de response (2e en 3e punt). Hier (3e punt) had je bijvoorbeeld ook kunnen zeggen: Toevoegen component "NaamPersoonInOnderzoek", waarmee boolean "aanduidingNaamGebruik" wordt toegevoegd aan inOnderzoek van de naam.
Bij 2e punt had je dan iets kunnen zeggen als "De property "adellijkeTitelPredikaat" (boolean) is toegevoegd aan "NaamInOnderzoek". Gevolg is dat "adellijkeTitelPredikaat" ook is toegevoegd aan "inOnderzoek" van de naam in resources "ouder", "kind" en "partner"." (zoals je het nu hebt gezegd suggereer je door de hoofdletter I in "InOnderzoek" dat dit een schemacomponent is, terwijl de beschrijving het denk ik heeft over de property (gegevensgroep) inOnderzoek in component "Naam")